tests/end_to_end/test_aws_infrastructure_monitoring/data/playbooks/teardown_playbook.yaml

```diff
@@ -0,0 +1,20 @@
+- name: Cleanup environment
```

Comment:
Please rename the playbook to `teardown.yaml`. Also, we need to delete the previously created bucket.

Reply:
Done in 2d8c31b

Reply:
@juliamagan the deletion of the bucket won't be possible because @jmv74211 asked me to delete it manually not via API.